packages/react-ui-components/src/SelectBox/selectBox.js

```diff
@@ -5,12 +5,14 @@ import {$get} from 'plow-js';
 import SelectBox_Option_SingleLine from '../SelectBox_Option_SingleLine';
 import mergeClassNames from 'classnames';
 import isEqual from 'lodash.isequal';
+import {styleConstants} from '@neos-project/build-essentials/src/styles';
 
 // TODO: document component usage && check code in detail
 export default class SelectBox extends PureComponent {
     static defaultProps = {
         options: [],
         optionValueField: 'value',
+        limitRenderedOptions: 0,
```

Comment:
Sorry for the late review, but is there no way to do that without introducing yet one more API prop to SelectBox? It's a beast the way it is now anyways, and I would love to not further complicate it.

Reply:
Can define a value or height, but thought at the Codesprint that it would be nice to be able to adjust that. No one had objections.

Reply:
Should we abandon the change or define a fix value?

Reply:
I'd set it to some sane default value and not make it configurable. But does it really need to be limited in height at all?

Reply:
Thinking more about it, as an editor I definitely wouldn't like SelectBox to be limited in height, it's actually nice that you see a lot of stuff:

![image](https://user-images.githubusercontent.com/837032/52334773-8d016400-2a11-11e9-99f4-b4902c6bf9ea.png)

Reply:
But in the creationDialog in can make sense

Reply:
Therefore we decided more or less in salzburg that a configuration would be a nice way. But I can also understand @dimaip who wants to keep the complexity as low as possible.

To be honest I don't know another way yet to make it dynamically work without extra configuration.

Reply:
Should we probably discuss that in the guildUX or directly on discuss. Or do you think that not many will join the discussion?

Reply:
You can give it a try, good idea to ask

Reply:
I know, it is one thing more to configure. *But:* From the UI/UX point of view it makes sense. Especially in the creation dialog. The current behavior isn't very user-friendly. If we don't want to have configurable, it would make sense, to have at least in the creation dialog a maximum height. @danielkestler @dfeyer: What do you think about that?

Reply:
@jonnitto what do you mean by current behavior: that it doesn't flow out or that it becomes too long if it does?
While naturally it should flow out of the dialog body, I'm not really sure if that would be a better UX for the selectbox to be limited in height. How does it help the editor? I personally would love to see as much content as possible.

Reply:
@jonnitto Yes, absolutely makes sense in the creation dialog. I don't know if we need a setting to limit the items – maybe it should depend on your browser size?

Reply:
@dimaip Main issue is that it doesn't flow out.

Reply:
@jonnitto well, that's not somethings that we are discussing, it definitely should fly out :) I just don't think we need any special setting for it.

Reply:
@jonnitto I think the easiest way to solve the problem is to set the height of the box to a sane default like in your screenshot (around 200px height). If you can set it to „dynamically“ adapt to the viewport height, fine, even better. A configuration option should be avoided if possible.

Reply:
Yep, exactly what I was trying to say :)